Shared/ShortTimeSpan.cs

```diff
@@ -12,7 +12,11 @@ namespace OpenTap
     /// </summary>
     struct ShortTimeSpan
     {
+       
         public enum UnitKind{
+            Days,
+            Hours,
+            Minutes,
```

Comment:
It seems like we dont actually use these extra cases, right? Maybe we should leave them out.